.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Resolves #ISSUE.
+
+**No (!) PRs should be merged without at least one review from a different team member of the Mondo Technical team.**
+
+## Documentation
```

Comment:
```suggestion
## Documentation
<!--- A common case for documentation is the addition of new `make` goals. Descriptions should be documented for new goals both in the (i) `help` command at the bottom of the `mondo-ingest.Makefile` and (ii) `docs/developer/workflows.md`. -->
```

Optionally could add a blurb about if it's a new source, follow steps in `docs/developer/add-new-source.md`, but that is not a frequent case.